package.json

```diff
@@ -32,7 +33,9 @@
     "react-i18next": "^11.14.0"
   },
   "devDependencies": {
+    "@playwright/test": "^1.44.1",
     "@types/lodash": "^4.14.178",
+    "@types/node": "^20.14.6",
```

Comment:
Let's also update engine req to Node 20? WDYT?

```
  "engines": {
    "node": ">=20"
  }
```

Reply:
I tried it, but everything breaks, as we're using nextjs 12.
I think we'll need to migrate to version 14 first 🤯 

I'm planning to do that soon, so I'm creating the tests to ensure we don't break anything moving forward

Reply:
Oh sorry, this is webapp 😆  You're right Mr